ubuntu/standard/Dockerfile.template

```diff
@@ -149,3 +154,21 @@ RUN apt-get update \
  && apt-get install -y --no-install-recommends \
     subversion \
  && rm -rf /var/lib/apt/lists/*
+
+# Download hosted tool cache
+ENV AGENT_TOOLSDIRECTORY=/opt/hostedtoolcache
+RUN azcopy --recursive --source https://vstsagenttools.blob.core.windows.net/tools/hostedtoolcache/linux --destination $AGENT_TOOLSDIRECTORY
```

Comment:
Looks like this brings down a bunch of versions of Python. Should we no longer install the python and python3 packages via apt-get? This will already make the image explode in size so trying to minimize the impact.

Reply:
@DavidStaheli / @chrisrpatterson -- what do you think here?  The Pythons installed from apt will be in PATH, so I am afraid of breaking people's builds.  None of the tool cache versions will be in PATH until you invoke the Use Python Version task.

Reply:
That's a reasonable point - let's leave them in for now.